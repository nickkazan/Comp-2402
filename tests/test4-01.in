Hello
Ham
LinkedList
List
Dog
Mark
March
Nick
Speaking
Speaks
Spark
