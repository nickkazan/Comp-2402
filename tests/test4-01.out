Hello
Ham
LinkedList
List
Dog
